### ISSUE TITLE
[Xanh mono] Temporary removal

### ISSUE THREAD

**Issue by yellow-type-foundry** [google/fonts#2857]
We would like to temporarily remove the [Xanh Mono](https://fonts.google.com/specimen/Xanh+Mono?query=xanh#standard-styles) typeface as soon as possible. We detected issues which would impact users as following:

[Inconsistent x-height and width]
[Lack of kerning pairs]
[Inconsistent letterforms and numerals design]
[A few symbols have different design]
[Need additional glyphs]

....

We are currently working on a major update and refinement version of the typeface to prepare for our Foundry’s launch early next year and we don’t want everyone to use different versions of this typeface to avoid confusion and inconsistency. 

If we don’t take it down, we afraid that there will be more users who might download this current snapshot with such issues and won’t get the update when we fix them.

We believe that the impact is extremely minimal at this moment, since the typeface is recently uploaded, we figure that it would be best to temporarily take it down before any damage is been done, and release it again early next year.

Please advice,

Thank you!

**Comment by RosaWagner** [comment]
We can't really remove anything because it would directly impact users, even if there is not much of them, even more if there is an update coming soon. The best is too update quickly without too much regression (no change of vertical metrics mostly).

Please understand that publishing on GF is a big process implicating many different people. 

When the update can be released, ping me in this issue.

**Comment by yellow-type-foundry** [comment]
Thank you for letting us know! 
We totally agree with Google Font's policies and will try our bests to comply with it. We will ping you again when the new collection is ready.

**Comment by yellow-type-foundry** [comment]
@RosaWagner We're coming close to the new updates, which includes many changes in letterform, overall style and more weights. I am just wondering if a massive update like that is acceptable?

**Comment by yellow-type-foundry** [comment]
Cool then! We keep the metrics just the same. Thank you!

**Comment by yellow-type-foundry** [comment]
@RosaWagner Could you please let me know the next steps to update the font? We've got the Glyphs file updated & new ttfs exported.

**Comment by RosaWagner** [comment]
Thanks, it's in my queue, I'll ship it :)

**Comment by yellow-type-foundry** [comment]
@RosaWagner Sorry I wasn't clear, we still keep the files local. So you mean we can just push the updates in the original PR and that's all?

**Comment by RosaWagner** [comment]
Yes, I will run a QA, build the font, and if it passes I ship.

**Comment by yellow-type-foundry** [comment]
Got it. Thanks, will let you know soon!

**Comment by yellow-type-foundry** [comment]
@RosaWagner Hi Rosa, is there anyway we can just display Yellow Type Foundry here and not putting our names on the front page? This looks like Yellow Type Foundry is collaborating with Lam Bao & Duy Dao and it might be a bit misleading.
![image](https://user-images.githubusercontent.com/40597471/113243885-0fc69680-92de-11eb-89b1-3493a2bc4c58.png)

**Comment by RosaWagner** [comment]
Yes it possible, we will do that with the update. Is the update ready ?

**Comment by RosaWagner** [comment]
@yellow-type-foundry is the update ready ?

**Comment by RosaWagner** [comment]
@yellow-type-foundry you have been teasing this update for a year now :) Is it ready in your repo ?

**Comment by RosaWagner** [comment]
Since no update on this, I close. Please open an issue only if there is an actual available upgrade.
